bundles/storage/org.eclipse.smarthome.storage.json/OSGI-INF/jsonstorage.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others. All rights reserved. This program and the accompanying 
+	materials are made available under the terms of the Eclipse Public License v1.0 which accompanies this distribution, and 
+	is available at http://www.eclipse.org/legal/epl-v10.html -->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" deactivate="deactivate"
+	immediate="true" name="org.eclipse.smarthome.storage.json">
+	<implementation class="org.eclipse.smarthome.storage.json.JsonStorageService" />
+
+    <property name="service.pid" type="String" value="org.eclipse.smarthome.storage.json" />
+	<property name="service.config.description.uri" type="String" value="system:json_storage" />
```

Comment:
I cannot find a config description for system:json_storage.
Anyhow, I don't think those settings need to be user adjustable through the UI - so I'd suggest to remove those service.config properties here.